check_tools/smartdiff.py

```diff
@@ -0,0 +1,52 @@
+import sys
+
+def format_map(black_list):
```

Comment:
An example of what kind of string format this is expecting would go a long way for readability and understanding what exactly this is parsing.